run_docker.sh

```diff
@@ -10,12 +10,18 @@ docker_image_name=$1
 rundir_host=$2
 fv3config_cache_dir=$3
 
-cp submit_job.sh $rundir_host
```

Comment:
why remove this step?

Reply:
If you want to delete the duplicate `submit_job.sh`, you can copy instead from `tests/pytest/run_files/submit_job.sh`.